src/styles/card.scss

```
@@ -0,0 +1,109 @@
+.catalog {
+  width: getWidth($width, $gap, $index);
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto;
+
+  &__card {
+    margin-right: 48px;
+
+    &:nth-child(-n+4) {
+      padding-top: 50px;
+      margin-bottom: 46px;
+    }
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+  }
+
+  &:nth-last-child(-n + 4) {
+    padding-bottom: 50px;
+  }
+}
+
+.card {
+  &__container {
+    width: 200px;
+    padding: 16px;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+  }
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 16px auto 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $main-color;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+    color: $second-color;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-color;
+  }
+
+  &__stars-block {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    line-height: 18px;
+    font-size: 12px;
+    color: $second-color;
+  }
+
+  &__price-item {
+    color: $main-color;
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+  }
+
+  &__price-block {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-top: 16px;
+    width: 166px;
+    height: 40px;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    color: #fff;
+    background: #00acdc;
+    border: 1px solid #00acdc;
+    border-radius: 5px;
+    text-decoration: none;
+    text-transform: uppercase;
```

would be better if you make an empty line between logic block styles
```suggestion
    display: flex;
    align-items: center;
    justify-content: center;
    
    margin-top: 16px;
    width: 166px;
    height: 40px;
    
    font-size: 14px;
    line-height: 16px;
    font-weight: 700;
    color: #fff;
    text-decoration: none;
    text-transform: uppercase;
    
    background: #00acdc;
    border: 1px solid #00acdc;
    border-radius: 5px;
```